wmfdata/hive.py

```diff
@@ -63,77 +65,23 @@ def run_cli(
     # error ourselves. However, there's no simple way to determine if multiple
     # queries have been passed or separate their output, so it's not worth
     # the effort.
-    merged_commands = ";\n".join(commands)
+    commands = ";\n".join(ensure_list(commands))
```

Comment:
It looks like you can't execute multiple commands this way when using a Python client. Instead, you'll want to loop through the commands, saving the output to a variable if it exists. Here's how we do it with Spark: https://github.com/wikimedia/wmfdata-python/blob/master/wmfdata/spark.py#L316

Reply:
hm, so that just returns the results from the last query or None.  Would we maybe just want to return an array of all the results formatted as requested?

Reply:
And I tested multiple commands, they seemed to work ok with my concatenation, did I miss something?  What commands failed for you?

Reply:
Hmm, what did you run?

I tried running 
```
import wmfdata as wmf

hive_db = "nshahquinn"
wmf.hive.run([
    f"DROP TABLE IF EXISTS {hive_db}.wmfdata_test_1",
    f"DROP TABLE IF EXISTS {hive_db}.wmfdata_test_2",
    f"DROP TABLE IF EXISTS {hive_db}.wmfdata_test_empty",
])
```

That gave me the following error:
> Error while compiling statement: FAILED: ParseException line 1:46 missing EOF at ';' near 'wmfdata_test_1'

Reply:
Oh, I see, you updated your message to say it _was_ failing. When I replied yesterday, I thought you were saying the opposite :joy:

Reply:
my bad, yeah, this is now fixed